PowerEditor/src/MISC/Common/FileInterface.cpp

```diff
@@ -33,40 +33,48 @@ Win32_IO_File::Win32_IO_File(const wchar_t *fname)
 {
 	if (fname)
 	{
-		generic_string fn = fname;
-		_path = std::string(fn.begin(), fn.end());
+		std::wstring fn = fname;
+		std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+		_path = converter.to_bytes(fn);
 		_hFile = ::CreateFileW(fname, _accessParam, _shareParam, NULL, _dispParam, _attribParam, NULL);
+
+
+		if (NppParameters::getInstance().doNppLogNulContentCorruptionIssue())
+		{
+			generic_string nppLogNetworkDriveIssueLog = TEXT("c:\\temp\\");

```

Comment:
Wouldn't it be easier just to use %UserProfile% that always exists? So the user is not obligated to create it beforehand.

Reply:
@mere-human 
Yes, indeed.
In the newest commit `%APPDATA%\Notepad++\` is used for storing the log files.